.changeset/dirty-bears-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+Adding functionality to rh-alert to allow the internal close button to set the host element to hidden and to bubble up a close event.
```

Comment:
```suggestion
Added behaviour to `<rh-alert>`'s close button: clicking it now removes the element.
```

Usually we don't need technical details like "this event bubbles" in the changelog, and they should be past-tense